File: app/src/main/java/com/callmangement/ui/pos_help/model/ModelFAQList.java
package com.callmangement.ui.pos_help.model;

import com.google.gson.annotations.SerializedName;

public class ModelFAQList {
    @SerializedName("id") private String id;
    @SerializedName("question") private String question;
    @SerializedName("answer") private String answer;
    boolean selectFlag = false;
    boolean visibleItemFlag = false;

    public String getId() {
        if (id == null)
            return "0";
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getQuestion() {
        if (question == null)
            return "";
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }

    public String getAnswer() {
        if (answer == null)
            return "";
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }

    public boolean isSelectFlag() {
        return selectFlag;
    }

    public void setSelectFlag(boolean selectFlag) {
        this.selectFlag = selectFlag;
    }

    public boolean isVisibleItemFlag() {
        return visibleItemFlag;
    }

    public void setVisibleItemFlag(boolean visibleItemFlag) {
        this.visibleItemFlag = visibleItemFlag;
    }
}

File: app/src/main/java/com/callmangement/ui/pos_help/model/ModelFAQList.kt
package com.callmangement.ui.pos_help.model

import com.google.gson.annotations.SerializedName

class ModelFAQList {
    @SerializedName("id")
    var id: String? = null
        get() {
            if (field == null) return "0"
            return field
        }

    @SerializedName("question")
    var question: String? = null
        get() {
            if (field == null) return ""
            return field
        }

    @SerializedName("answer")
    var answer: String? = null
        get() {
            if (field == null) return ""
            return field
        }
    var isSelectFlag: Boolean = false
    var isVisibleItemFlag: Boolean = false
}
